tests/helpers.py

```diff
@@ -710,6 +723,27 @@ def get_cephfs_name():
     return result['items'][0].get('metadata').get('name')
 
 
+def pull_images(image_name):
+    """
+    Function to pull images on all nodes
+
+    Args:
+        image_name (str): Name of the container image to be pulled
+
+    Returns:
+        bool: True if deletion of image pull is successful
+
+    """
+
+    node_objs = node.get_node_objs(get_worker_nodes())
+    for node_obj in node_objs:
+        logging.info(f'pulling image "{image_name}  " on node {node_obj.name}')
+        assert node_obj.ocp.exec_oc_debug_cmd(
+            node_obj.name, cmd_list=[f'podman pull {image_name}']
+        )
+    return
```

Comment:
Just a return statement with no value here won't return true, Can we change here ? I don't think we need a return statement here, Do we need ?

Reply:
Is there any difference? I don't see any. 

```
>>> def fn1():   
... 	return 
... 
>>> def fn2():
... 	pass
... 
>>> a = fn1()
>>> a
>>> type(a)
<class 'NoneType'>
>>> b = fn2()
>>> b
>>> type(b)
<class 'NoneType'>
```
I'm not sure what is the best practice here, so went with a similar function in ocs-ci.

Reply:
Best practice is ```return return_value```. If the function do not have a return value, no need to write return statement. So the return can be removed. Only return does not serve any purpose.

Reply:
Okay, removed `return` statement and fixed docstring.